docker-compose.test.yml

```diff
@@ -30,8 +30,8 @@ services:
       service: base
     environment:
       - APP_SQLALCHEMY_DATABASE_URI=postgresql+psycopg2://inspirehep:dbpass123@test-database:5432/inspirehep
-      - APP_BROKER_URL=amqp://guest:guest@test-rabbitmq:5672//
-      - APP_CELERY_RESULT_BACKEND=amqp://guest:guest@test-rabbitmq:5672//
+      - APP_CELERY_BROKER_URL=pyamqp://guest:guest@test-rabbitmq:5672//
+      - APP_CELERY_RESULT_BACKEND=rpc://guest:guest@test-rabbitmq:5672//
```

Comment:
why not redis?

Reply:
Redis is a much more sensible choice for a result backend, but I'm thinking that that migration requires some care to be performed without data loss on PROD. So I'd propose to skip that for now, and just bump Celery in this PR.

Reply:
...it turns out that PROD has been using Redis as a result backend since immemorable times, so let me add a commit that does the same for DEV.